docs/topic-guides/blendable-web-apps/customizing-solutions-design.md

```diff
@@ -0,0 +1,92 @@
+# Customizing Solution's Design
+
+
+Building your app or solution design the way we suggest does not only help you create a consistent look and feel fast and easily. It also gives you means to customize it later, by applying global themes, switching color & configuration schemes, or replacing the definition of every component at once.
```

Comment:
Prefer "and" over "&"

Try to write the way that you speak, so "It also lets you" over "It also gives you means"